### DISCUSSION TITLE
EvolutionAPI - An open-source WhatsApp integration API

### DISCUSSION THREAD

**Discussion by louanfontenele** [coollabsio/coolify#4548] [Service Template Requests]
Evolution API is an open-source WhatsApp integration API

https://github.com/EvolutionAPI/evolution-api

**Comment by ikx94** [comment]
Would be a great addition!

**Comment by djalmaaraujo** [comment]
@louanfontenele @ikx94 https://github.com/coollabsio/coolify/pull/5302

**Comment by berlinseo** [comment]
That’s a great project. If you’re creating a space around EvolutionAPI for WhatsApp integrations, it could also be useful to discuss platforms that already provide ready-to-use solutions built on the WhatsApp API.

For example, tools like Getgabs, GetItSMS, WATI, Gupshup, and AiSensy are commonly used for WhatsApp automation, bulk messaging, and chatbot workflows. These platforms help businesses manage conversations, automate replies, and run campaigns without building everything from scratch.

It might be interesting to create a discussion space where developers can share how APIs like EvolutionAPI compare with or integrate alongside these kinds of tools and platforms. That could help people decide whether they want a custom API-based setup or a ready-made WhatsApp automation platform.
